buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -231,7 +232,7 @@ public void afterExecute(Task task, TaskState state) {
                             .filter(entry -> runningClusters.contains(entry.getKey()))
                             .map(Map.Entry::getKey)
                             .forEach(each -> {
-                                each.stop(false);
+                                stopCluster(each, false);
```

Comment:
Same as above, let's avoid naming `forEach()` arguments `each` as it's a bit confusing.